frontend/src/v5/validation/shared/validators.ts

```diff
@@ -55,7 +55,7 @@ export const uploadFile = Yup.mixed().nullable().test(
 	({ size }) => size && (size < ClientConfig.uploadSizeLimit),
 );
 
-export const alphaNumericHyphens = /^[\w|_|-]*$/;
+export const alphaNumericHyphens = /^[\w|-]*$/;
```

Comment:
I think the "|" is treated as a valid symbol instead of an "or", so `ale|dan` would be valid if tested against this. Without opening a new issue for it, can you fix it here please?